DSharpPlus/DiscordIntents.cs

```diff
@@ -168,13 +168,25 @@ public enum DiscordIntents
     /// </summary>
     AutoModerationExecution = 1 << 21,
 
+    /// <summary>
+    ///  Whetever to include add and remove of a poll votes events in guilds.
+    /// <para>This includes <see cref="DiscordClient.MessagePollVoted"/></para>
+    /// </summary>
+    GuildMessagePolls = 1 << 24,
+
+    /// <summary>
+    ///  Whetever to include add and remove of a poll votes events in direct messages.
+    /// <para>This includes <see cref="DiscordClient.MessagePollVoted"/></para>
+    /// </summary>
+    DirectMessagePolls = 1 << 25,
+
     /// <summary>
     /// Includes all unprivileged intents.
     /// <para>These are all intents excluding <see cref="DiscordIntents.GuildMembers"/> and <see cref="DiscordIntents.GuildPresences"/>.</para>
     /// </summary>
     AllUnprivileged = Guilds | GuildModeration | GuildEmojisAndStickers | GuildIntegrations | GuildWebhooks | GuildInvites | GuildVoiceStates | GuildMessages |
                       GuildMessageReactions | GuildMessageTyping | DirectMessages | DirectMessageReactions | DirectMessageTyping | ScheduledGuildEvents |
-                      AutoModerationEvents | AutoModerationExecution,
+                      AutoModerationEvents | AutoModerationExecution /*| GuildMessagePolls | DirectMessagePolls*/,
```

Comment:
Upon adding the [documented intents](https://github.com/discord/discord-api-docs/pull/6746), currently receiving `Connection terminated (4013, 'Invalid intent(s).')`, might not be rolled out yet? Commented out for the time being for testing.
```
GUILD_MESSAGE_POLLS (1 << 24)
  - MESSAGE_POLL_VOTE_ADD
  - MESSAGE_POLL_VOTE_REMOVE
DIRECT_MESSAGE_POLLS (1 << 25)
  - MESSAGE_POLL_VOTE_ADD
  - MESSAGE_POLL_VOTE_REMOVE
```

Reply:
It was supposed to be deployed today, but still hasn't, will try again tomorrow.